SimG4CMS/HcalTestBeam/test/python/run2006_cfg.py

```diff
@@ -173,6 +181,7 @@
 
 process.g4SimHits.SteppingAction = cms.PSet(
     process.common_maximum_timex,
+    MaxNumberOfSteps        = cms.int32(50000),
```

Comment:
There is not need to reduce the default value.

Reply:
This is the default value but since SteppingAction is redefined here, I have to add this